### PR TITLE
add aduh95 to Moderation Team in policy doc

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

**Comment by Trott** [comment]
@nodejs/moderation

**Comment by ljharb** [comment]
Could this be automated with ncu-team-sync and GitHub actions?

**Comment by Trott** [comment]
> Could this be automated with ncu-team-sync and GitHub actions?

Yes, I believe so. Separate PR (perhaps from someone else)?

**Review by targos — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

File: Moderation-Policy.md [hunk 1]
```diff
@@ -267,6 +267,8 @@ remove resigning team member from respective permissions and private access.
 <!-- referenced from the CoC page -->
 <a id="current-members"></a>
 ### Current Members of Moderation Team
+* [aduh95](https://github.com/aduh95) -
+  **Antoine du Hamel** <<duhamelantoine1995@gmail.com>> (he/him)
```
Comment (richardlau) [inline Moderation-Policy.md:271]:
```suggestion
**Antoine du Hamel** <<duhamelantoine1995@gmail.com>> (he/him)
```
for consistency with the other entries.

**Review by richardlau — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Review by aduh95 — APPROVED** [review]

**Comment by Trott** [comment]
> > Could this be automated with ncu-team-sync and GitHub actions?
> 
> Yes, I believe so. Separate PR (perhaps from someone else)?

@ljharb `ncu-team sync` will not include the email address of the team member. In this context, I think the email address is important, so we'd need to update node-core-utils to allow that as an option first.

**Pull request merged**